### DISCUSSION TITLE
Better onboarding to Slack?

### DISCUSSION THREAD

**Discussion by paulcretu** [orcasound/.github#17] [General]
I want to invite someone to the Orcasound Slack, and I just noticed that we don't have a Slack invite listed on http://orcasound.net. The closest thing I was able to find is a link (but not an invite) buried halfway down the support page (https://www.orcasound.net/support/). 

The good news is that there is indeed an invite link on this github org page (https://github.com/orcasound), but it's in the middle of a lot of text.

Ideally I would send someone a page that tells them how to join Slack, but it doesn't currently exist. Currently, Slack is central to Orcasound collaboration, and unless that changes, this seems critical to fix.

It's okay, I'll just send them the invite link directly for now, but how can we make it easier to discover? Where are all the current onboarding points located?

**Comment by paulcretu** [comment]
Here are all the entry points I know of and if they work:
- ✅ [Github org page](https://github.com/orcasound)
- ✅  Badges at the top of [orcasite](https://github.com/orcasound/orcasite) and [orcahome](https://github.com/orcasound/orcahome) README
- ❌ Link on [DemocracyLab](https://www.democracylab.org/projects/81) profile page
- ❌ [Volunteer handbook](https://drive.google.com/file/d/1GHuylE3qPm2UAyUA_f1yOtMgutBpXF6V/view)
- ✅ [Product Wiki Onboarding page](https://github.com/orcasound/product/wiki/1.-Orcasound-UX-Team-Onboarding) (credit @saraloup)

Missing any? Please let me know. These aren't necessarily places where it's easy to find the invite link, but it does exist. ~~I haven't tested if all the links still work.~~

UPDATE: None of the links were working, all of them expired. I've fixed the ones on github and marked working/not.

Notable places we don't have Slack onboarding but we probably should:
- https://orcasound.net
- https://live.orcasound.net
- Every project README
- [Trello board](https://trello.com/b/wBg0qhss/orcasound-roadmap)
- Notification emails

**Reply by saraloup** [reply]
@paulcretu There is a Slack link on the [Product Wiki Onboarding page](https://github.com/orcasound/product/wiki/1.-Orcasound-UX-Team-Onboarding).

Additionally I could add Slack in the email template footer if we want but is Slack beneficial for anyone interested in whale conservation or only those volunteering with the tech? I'm thinking people might think Slack is a group for more general chatting about the whales and from my perspective there isn't much along general conservation to be worth joining if I'm not volunteering but maybe we want to change that?

**Reply by paulcretu** [reply]
Thanks @saraloup! Added to the list.

> is Slack beneficial for anyone interested in whale conservation or only those volunteering with the tech?

Great question. Yes, maybe the email notifications aren't the right place. I think there's a balance between keeping the call to action clear, but at the same time leaving a path towards getting involved. I wonder how many subscribers even know they can get involved?

It doesn't need to be a direct link to Slack, a simplified landing page would do (I know there's a redesigned "get involved" page in Orcahome, I haven't seen the latest designs).

On the other hand, it's been brought up before that we should have a live chat for listeners. Slack hasn't felt like the right tool, but it could be a good time to revisit:

- We may want to switch away from Slack ([discussion](https://github.com/orgs/orcasound/discussions/16))
- We're just now adding user accounts for listeners (https://github.com/orcasound/orcasite/issues/303)
- There's [a lot of interest](https://apnews.com/article/orcas-killer-whales-whatsapp-facebook-salish-wildlife-0e86cd0a5403049a74386d66fc0ae8c8) in listening and chatting on other platforms!

**Reply by scottveirs** [reply]
@saraloup By "email template" did you mean the email that folks like @UXBrendan and you have been sending to new volunteers, e.g. in response to the automated new volunteer emails we get from DemocracyLab? Or did you mean the emails we send to Orcasound subscribers to announce a live-listening opportunity? (I think you meant the former, but infer that @paulcretu thought you meant the latter?)

**Reply by saraloup** [reply]
@scottveirs No, I was referring to the subscriber live-listening event [email template](https://github.com/orcasound/product/issues/66) I'm working on so I think @paulcretu and I are on the same page. I do think keeping the link on a "Get Involved" page or Github (which is linked in the live event emails) would be most appropriate. Although you could always make "get involved with Orcasound" one of the sender input conservation calls to action in the live event email to bring more awareness to to tech savvy subscribers that may want to help out with design or dev.

I'm not sure what's been discussed about onboarding. I do think all the open links make it easy to join but could also be leading to the numerous posts of new volunteers asking how to get involved (well, when the links were working). This can create more work for people to refer them to the Product Wiki @UXBrendan  wrote or the [onboarding website](https://sites.google.com/view/orcasound-onboarding-site/home) Jennifer worked on (**another Slack link** is in the footer there- placeholder right now). If keeping Slack, maybe we could direct all links to that onboarding site so that volunteers can read about the projects and how to get involved first before clicking through to Slack. Or since I think the majority of volunteers come through DemocracyLab maybe you don't want to have an open link there so they receive the email Brendan sends in response. I am fairly certain I joined from that link to check it out but did not sign up through DemocracyLab so I didn't get an email. So then I had no idea what was going on or how to get involved for a while.

**Comment by saraloup** [comment]
@paulcretu I came across another Slack link that appears expired under the [volunteer section here](https://github.com/orcasound/aifororcas-livesystem/wiki/Moderation).
